src/ansys/dpf/core/config.py

```diff
@@ -34,6 +34,24 @@ class Config:
         Server with the channel connected to the remote or local instance. The default is
         ``None``, in which case an attempt is made to use the global server.
 
+    Examples
+    --------
+    Create an operator configuration, instantiate the operator and
+    run it with the configuration.
+
+    >>> from ansys.dpf import core as dpf
+    >>> config_add = dpf.Config("add")
+    >>> config_add.set_work_by_index_option(True)
+    >>> op = dpf.operators.math.add(config=config_add)
+
+    Modify the copy of an operator's configuration and set it back.
```

Comment:
By set it back, do you mean reset it? Or set it to something else?

Reply:
@JennaPaikowsky it means "replace the current operator's config by the modified copy of it"; would you have any suggestion for that ?